keras_cv/models/object_detection/faster_rcnn/faster_rcnn.py

```diff
@@ -12,217 +12,47 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import tensorflow as tf
+import copy
+
+import tree
 from absl import logging
-from tensorflow import keras
 
 import keras_cv
 from keras_cv import bounding_box
 from keras_cv import layers as cv_layers
+from keras_cv.api_export import keras_cv_export
+from keras_cv.backend import keras
+from keras_cv.backend import ops
 from keras_cv.bounding_box.converters import _decode_deltas_to_boxes
+
+# All the imports from legacy
 from keras_cv.bounding_box.utils import _clip_boxes
 from keras_cv.layers.object_detection.anchor_generator import AnchorGenerator
 from keras_cv.layers.object_detection.box_matcher import BoxMatcher
 from keras_cv.layers.object_detection.roi_align import _ROIAligner
 from keras_cv.layers.object_detection.roi_generator import ROIGenerator
 from keras_cv.layers.object_detection.roi_sampler import _ROISampler
 from keras_cv.layers.object_detection.rpn_label_encoder import _RpnLabelEncoder
-from keras_cv.models.object_detection import predict_utils
```

Comment:
By the way, this is no longer used after this PR, and we should get rid of predict_utils entirely